fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -73,6 +73,7 @@
 import static org.fcrepo.kernel.api.RdfLexicon.HAS_PRIMARY_TYPE;
 import static org.fcrepo.kernel.api.RdfLexicon.INBOUND_REFERENCES;
 import static org.fcrepo.kernel.api.RdfLexicon.INDIRECT_CONTAINER;
+import static org.fcrepo.kernel.api.RdfLexicon.JCR_NAMESPACE;
```

Comment:
I agree that the testing is a little weird. An alternative is to just trust that the testing at the fcrepo-kernel-modeshape level is complete, thereby removing theses sorts of tests from the fcrepo-http-api module. For now, though, I'd prefer to keep these tests in place to guarantee that those JCR types aren't escaping.